packages/amplify-graphql-schema-generator/vpc-db-lambda/src/connection.ts

```diff
@@ -37,5 +37,6 @@ export const establishDBConnection = (config: any): any => {
 const getRDSCertificate = (): string => {
   // This certificate file is copied from the parent folder `amplify-graphql-schema-generator/certs/aws-rds-global-bundle.pem`.
   const RDS_CERT_FILE_NAME = 'aws-rds-global-bundle.pem';
-  return fs.readFileSync(RDS_CERT_FILE_NAME, 'utf-8');
+  const certPath = `${__dirname}/certs/${RDS_CERT_FILE_NAME}`;
```

Comment:
Is this going to cause any problems on Windows where `__dirname` may have backslashes instead of forward slashes? I understand that Node uses forward slashes transparently; I'm asking specifically about a mixed-mode path. Why not `path.join`?

Reply:
Updated.